etl/steps/data/garden/faostat/2023-02-22/additional_variables.meta.yml

```diff
@@ -626,3 +626,45 @@ tables:
           For this reason we look at decadal changes in agricultural land. We look at land use in the latest year relative to 10 years before.
 
           If land use is lower in the latest year then we suggest that land use may have peaked. If land use it the same or higher than a decade back, we suggest that it hasn't, or this is uncertain.
+  hypothetical_meat_consumption:
+    variables:
+      animals_global:
+        title: Number of slaughtered animals to produce meat worldwide
+        unit: "animals"
+        short_unit: ""
+      animals_global_hypothetical:
+        title: Hypothetical number of slaughtered animals if everyone ate like the average citizen of a given country
+        unit: "animals"
+        short_unit: ""
+        description: |
+          Hypothetical number of slaughtered animals worldwide if everyone in the world ate the same quantity as the average citizen of a given country.
+
+          This is a hypothetical variable derived by Our World in Data which answers the question: "how many animals would need to be slaughtered if everyone in the world was to consume the average per capita amount of a given country?". For example "how many animals would need to be slaughtered if everyone in the world consumed the same amount of meat as the average UK citizen?".
```

Comment:
Unsure about it, but should the question start with caps? 

> "How many...

Reply:
I agree, done.